package.json

```diff
@@ -64,6 +66,7 @@
     "lb_pool": "^1.4.0",
     "ldjson-stream": "^1.2.1",
     "loadtest": "1.2.14",
+    "logtron": "^8.11.1",
```

Comment:
Larch's logtron backend requires `logtron/entry`. This is probably wrong. Maybe I should just copy that class def into Larch.